src/amo/components/HomepageShelves/index.js

```diff
@@ -0,0 +1,82 @@
+/* @flow */
+import * as React from 'react';
+import config from 'config';
+
+import translate from 'amo/i18n/translate';
```

Comment:
Nit: We try to keep these in alphabetical order, based on whence the import is from, so the import from `amo/i18n/translate` would come after the import from `amo/constants`, because `constants` is before `i18n` in the alphabet.